src/main/scala/com/kinja/amqp/model/MessageConfirmation.scala

```diff
@@ -1,9 +1,11 @@
 package com.kinja.amqp.model
 
-import java.util.UUID
+import java.util.Date
 
 case class MessageConfirmation(
 	id: Option[Long],
-	channelId: UUID,
+	channelId: String,
 	deliveryTag: Long,
-	multiple: Boolean)
+	multiple: Boolean,
+	createdTime: Date
```

Comment:
can I ask why this field is a Date? why not just a Long for timestamps? Date is a really clunky Java thing in my opinion